otterdog/jakartaee.jsonnet

```diff
@@ -1782,6 +1782,24 @@ orgs.newOrg('ee4j.jakartaee-platform', 'jakartaee') {
           requires_status_checks: false,
           requires_strict_status_checks: true,
         },
+        orgs.newBranchProtectionRule('1.1') {
```

Comment:
any chance we can do it as a pattern? 
seems like it should be a repository rule in that case though (https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-rulesets/about-rulesets#branch-and-tag-rulesets) 

*I'm just thinking, since new release branches will be added in the future, we probably wouldn't want to update the config each time